templates/macros.j2

```diff
@@ -51,10 +51,10 @@ GITHUB_CONTEXT: "{{ '${{ github.event.pull_request.commits_url }}' }}"
 - name: "Install python dependencies"
   run: |
     echo ::group::PYDEPS
-    pip install {{ deps | join(" ") }}
-    {%- if 'httpie' in deps %}
+    pip install {{ deps | map("shquote") | join(" ") }}
+    §= if 'httpie' in deps
```

Comment:
I've never seen § used in templates.

Reply:
https://jinja.palletsprojects.com/en/3.0.x/templates/#line-statements

I thought using "#" has some detrimental effects when heavily templating shell scripts. OTOH I believe using line statements will reduce time we need to think about these whitespace eating `-` a lot.

Reply:
Thank you! neat feature